### ISSUE TITLE
TypeError: Cannot read property 'substring' of undefined

### ISSUE THREAD

**Issue by vasttian** [tinymce/tinymce#5144]
**What is the current behavior? Describe the bug**

This error occurs when I use tinymce,
 **TypeError: Cannot read property 'substring' of undefined**.
I found out because I used a `script` tag without `src` on the page.
I think this is a common situation.
I found the code in the [tinymce/modules/tinymce/src/core/main/ts/api/EditorManager.ts](https://github.com/tinymce/tinymce/blob/1c3d4775f4ca0d8c63afbc30c4f909870ec573ae/modules/tinymce/src/core/main/ts/api/EditorManager.ts#L248-L256) path file, 

L250: `src = scripts[i].src;` 
Can it be modified to `src = scripts[i].src || '';` ?

Thx.

![image](https://user-images.githubusercontent.com/17924984/64153801-28509f80-ce62-11e9-9012-3922451ddc65.png)

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

**What is the expected behavior?**

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

"tinymce": "^5.0.15",

**Comment by EkimChau** [comment]
Hi Vasttian

Thanks for posting, we'll take this on as an improvement task.  May I ask about the condition where script tags are missing the src attr? Are these leftover artefacts from other scripts?  I'd like to understand the situations were this could occur to better improve the robustness of tinymce.

Ekim

**Comment by vasttian** [comment]
Hi Ekim,

I used an animated svg icon.

Like this:
```javascript
<svg>
  <g>
    <script>
      !function(){}();
    </script>
  </g>
</svg>
```

**Comment by lnewson** [comment]
This issue has been fixed in TinyMCE 5.0.16, which has just been released to the community. Please let us know if you have any further issues.
